diff -urN ./CMakeLists.txt ../visp3.0_base/CMakeLists.txt
--- ./CMakeLists.txt	2016-12-23 18:56:11.852505167 -0300
+++ ../visp3.0_base/CMakeLists.txt	2016-12-23 18:56:29.012414375 -0300
@@ -720,5 +720,3 @@
 if(BUILD_TUTORIALS)
   add_subdirectory(tutorial)
 endif()
-
-find_package(rock)
diff -urN ./manifest.xml ../visp3.0_base/manifest.xml
--- ./manifest.xml	2016-12-23 18:56:11.852505167 -0300
+++ ../visp3.0_base/manifest.xml	1969-12-31 21:00:00.000000000 -0300
@@ -1,10 +0,0 @@
-<package>
-    <description brief="VISP is an Open Source Visual Servoing Plataform, developed by Inria Lagadic team."></description>
-  <author>Inria Lagadic team</author>
-  <maintainer>Diego Cesar/diego.b.cesar@gmail.com</maintainer>
-  <license></license>
-  <url>http://</url>
-  <logo>http://</logo>
-  <depend package="base/cmake" />
-  <depend package="base/types" />
-</package>
diff -urN ./visp3.pc.in ../visp3.0_base/visp3.pc.in
--- ./visp3.pc.in	2016-12-23 18:56:11.852505167 -0300
+++ ../visp3.0_base/visp3.pc.in	1969-12-31 21:00:00.000000000 -0300
@@ -1,11 +0,0 @@
-prefix=@CMAKE_INSTALL_PREFIX@
-exec_prefix=@CMAKE_INSTALL_PREFIX@
-libdir=${prefix}/lib
-includedir=${prefix}/include
-
-Name: @TARGET_NAME@
-Description: @PROJECT_DESCRIPTION@
-Version: @PROJECT_VERSION@
-Requires: @PKGCONFIG_REQUIRES@
-Libs: -L${libdir} -l@TARGET_NAME@ @PKGCONFIG_LIBS@
-Cflags: -I${includedir} @PKGCONFIG_CFLAGS@
